stream_alert/rule_processor/rules_engine.py

```diff
@@ -385,6 +388,24 @@ def process(cls, input_payload):
                                                    payload.normalized_types,
                                                    rule.datatypes)
                     record['normalized_types'] = types_result
+
+                    # If Threat Intel feature is enabled, it will detect if any
+                    # data in the record matching to known IOCs. If there is a
+                    # matching, IOC information will be added to record and sent
+                    # to outputs.
+                    if threat_intel_config and threat_intel_config.get('enabled'):
+                        ioc_result, ioc_type, ioc_value = cls.is_ioc(
+                            record,
+                            threat_intel_config.get('mapping'))
+
+                        # if there is ioc mathing, append this ioc_type and
+                        # ioc_value to the record which will be sent to outputs
+                        if ioc_result:
+                            record['streamalert_ioc'] = {
```

Comment:
use `streamalert:ioc` to be consistent with other `streamalert` appended keys (as mentioned in my previous comment)

Reply:
Yes, I tried to use `streamalert:ioc` in the beginning, and I do agree we should keep naming consistent. But I encountered strange situation that key `streamalert:ioc` can not be inserted to the record unless I have backslash before colon, `streamalert\:ioc`. However, pylint complaint `Anomalous backslash in string`. So I gave up and used `streamalert_ioc` instead.